regression/scripts/segments/single_point_segment_test.py

```diff
@@ -0,0 +1,266 @@
+# single_point_segment_test.py
```

Comment:
I will combine the scripts. Since we are only testing the mission methods, there is no reason to save/load data. 

- The reason I separated the single point and other segments was because I had a problem with saving and loading results using the SUAVE Input-Output functions. There was a json encoder error. Since I wanted to user to be able to plot the differences I have to store the data. If you do not think this is necessary I can combine into one file and just compare truth values.